REST/src/Category.java

```diff
@@ -0,0 +1,52 @@
+

```

Comment:
@VasylSamuliak please pull develop branch and add data classes into initial project